src/main/java/seedu/tp/flashcard/FlashcardList.java

```diff
@@ -5,21 +5,34 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
 
 /**
  * List of flashcards.
  */
 public class FlashcardList {
+
+    private static Logger logger = Logger.getLogger(FlashcardList.class.getName());
     private List<Flashcard> flashcards;
 
     /**
      * Constructor for FlashcardList.
      */
     public FlashcardList() {
-        this.flashcards = new ArrayList<Flashcard>();
+        setupLogger();
+        this.flashcards = new ArrayList<>();
     }
 
+    /**
+     * Constructor for FlashcardList.
+     *
+     * @param flashcardList the list of flashcards to be added
+     */
     public FlashcardList(List<Flashcard> flashcardList) {
+        setupLogger();
```

Comment:
We can replace line 35 and 36 here with a call to this()

Reply:
Updated :)